sql/glam-fenix-dev/glam_etl/org_mozilla_fenix__clients_daily_scalar_aggregates_startup_timeline_v1/query.sql

```diff
@@ -63,6 +63,111 @@ unlabeled_metrics AS (
         'sum',
         sum(CAST(metrics.counter.startup_timeline_clock_ticks_per_second AS INT64))
       ),
+      (
```

Comment:
I just wanted to confirm I remember this correctly. All these generated `query.sql` files that are being updated in this commit are just added for reference, right? But in the actual airflow job they are produced on the fly before running?

Reply:
Yep, these checked in files are mostly for being able to diff changes. The actual jobs will generate the sql from scratch to get the new probe names.